src/components/Modals/MintItemsModal/MintItemsModal.tsx

```diff
@@ -103,8 +107,8 @@ export default class MintItemsModal extends React.PureComponent<Props, State> {
                   {t('global.cancel')}
                 </Button>
               ) : (
-                <Button primary onClick={this.handleMintItems} loading={isLoading}>
-                  {t('global.done')}
+                <Button primary onClick={this.handleMintItems} loading={isLoading} disabled={this.isDisabled()}>
```

Comment:
Maybe we can move `this.isDisabled()` before the render's _return_ to just use variables for components